docs/user/dql/functions.rst

```diff
@@ -2096,6 +2096,45 @@ Example::
     | 3                             |
     +-------------------------------+
 
+SEC_TO_TIME
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: sec_to_time(number) returns the time in HH:mm:ss format.
```

Comment:
Mention that function round around 24 clock. Value equal to 25x3600 produce the same result as 3600.